src/docs/python-dependencies.mdx

```diff
@@ -21,10 +21,11 @@ So here are the rules we worked with so far:
 In order to avoid ever breaking master, you can:
 
 1. Open PR in getsentry to update requirements
-2. Open PR in sentry with the same branch name, put [`#sync-getsentry`](/workflow/) in PR description
+2. Open PR in sentry with the same branch name, put [`#sync-getsentry`](/workflow/) in the PR's description
+   - You can also add it in any other comment, however, you will need to push new changes to trigger the bot
```

Comment:
@armenzg Double-checked, the bot looks at the PR body only: https://github.com/getsentry/sentry-deploy-sync-hook/blob/master/deployhook.py#L170
So the tag has to go to the PR description (body) specifically, sorry for the confusion.